kv/kv.go

```diff
@@ -366,3 +366,9 @@ type SplitableStore interface {
 	WaitScatterRegionFinish(regionID uint64, backOff int) error
 	CheckRegionInScattering(regionID uint64) (bool, error)
 }
+
+// Used for pessimistic lock wait time
+var (
```

Comment:
A gentle remind, better code style here is defining a type like this:

```
type LockWaitType int
const(
    LockAlwaysWait LockWaitType = iota
    LockNoWait 
)
```

Reply:
Roger. the `uint64` type is used as input param type in `doLockKeys`, here to define 2 special case using this param, so they are not defined like above